massgov/openmass
# DP-35232 fix org page alerts position

Fixed:
  - description: Fixed alerts positioning on org pages.
    issue: DP-35232

Before:
![Screenshot 2024-10-01 at 1 36 10 PM](https://github.com/user-attachments/assets/418bae97-ada1-4709-aea3-41ad3076252e)

After:
![Screenshot 2024-10-01 at 1 35 59 PM](https://github.com/user-attachments/assets/23c39c0c-e980-460d-9901-de12fac67460)



To Test:
- Review alerts positioning on mobile and desktop https://pr2703-eibdrb4obs90nbx1ecn8cszojjtdtujs.tugboatqa.com/orgs/qag-organization-boards 
- Check other pages to confirm no other page types are affected, for example https://pr2703-eibdrb4obs90nbx1ecn8cszojjtdtujs.tugboatqa.com/how-to/qag-request-help-with-a-computer-problem
